libs/gltfio/src/AssetLoader.cpp

```diff
@@ -665,6 +655,7 @@ bool FAssetLoader::createPrimitive(const cgltf_primitive* inPrim, Primitive* out
                 .generateTrivialIndices = false,
                 .generateDummyData = false,
                 .generateTangents = true,
+                .sparseAccessor = (bool) inputAccessor->is_sparse,
```

Comment:
Just curious, why is the cast to `bool` necessary?

Reply:
Good question: cgltf_bool is an int because it's a pure C library.